NarulaAman/NarulaAman-Montreal.md

```diff
@@ -0,0 +1,5 @@
+# Montreal 
+
+### Restaurants
```

Comment:
@NarulaAman There are some good restaurants in Old Montreal but I don't remember their names.  Do you have any recommendations in Old Montreal?